### ISSUE TITLE
`CODEGEN_INTERFACE_PREFIX` not respected on `modelType` in `graphql-codegen-typescript-template`

### ISSUE THREAD

**Issue by KeithGillette** [dotansimha/graphql-code-generator#548]
When generating via `graphql-codegen-typescript-template` with a `CODEGEN_INTERFACE_PREFIX` set, the GraphQL operation response interface does not include the specified prefix, so it does not match the type assigned to the `operationType` field.

(Since the response interface is contained within a TypeScript namespace along with the `Variables` and `Query` or `Mutation` types, I'd recommend that it just be called `Response` or `ResponseData` instead of a calculated name based on the GraphQL operation name, anyway.)

**Comment by dotansimha** [comment]
@KeithGillette any chance you can add an example?

**Comment by KeithGillette** [comment]
Do you mean a reproduction of the bug or an example of my recommendation for response interface naming?

**Comment by dotansimha** [comment]
a reproduction or an example for input and output for the codegen

**Comment by KeithGillette** [comment]
Running the following command:

`CODEGEN_INTERFACE_PREFIX="I" CODEGEN_RESOLVERS=false node packages/graphql-codegen-cli/dist/cli.js --template graphql-codegen-typescript-template --schema ./dev-test/star-wars/schema.json --out ./dev-test/star-wars/types.ts "./dev-test/star-wars/**/*.graphql"`

produces the following excerpted output in `dev-test/star-wars/types.ts`:

```
...
export namespace CreateReviewForEpisode {
  export type Variables = {
    episode: IEpisode;
    review: IReviewInput;
  };

  export type Mutation = {
    __typename?: 'Mutation';
    createReview?: ICreateReview | null;
  };

  export type CreateReview = {
    __typename?: 'Review';
    stars: number;
    commentary?: string | null;
  };
}
...
```

Note that `CreateReview` !== `ICreateReview`

Reviewing the full output reveals that the problem appears to be broader than I first realized: Nothing generated as a TypeScript `type` gets the interface prefix (which makes sense, I suppose) but the interface prefix _is_ mistakenly applied when those types are used to type Query or Mutation parameters or responses.

So my initial recommendation to use the name `NameSpace.Response` or `NameSpace.ResponseData`, while I think still sound for simplicity and consistency with `NameSpace.Variables`, would not fix the entire problem.

**Comment by dotansimha** [comment]
Fixed in 0.14.
